ringpop.go

```diff
@@ -246,6 +258,8 @@ func (rp *Ringpop) stopTimers() {
 		for ticker := range rp.tickers {
 			ticker.Stop()
 		}
+		close(rp.tickersStop)
+		rp.tickersStop = nil
```

Comment:
This line causes a data race with lines 229 and 247. Is it really needed to set it to `nil` when the timers are stopped?